### DISCUSSION TITLE
Component upgrades- Fleet / Wazuh etc.

### DISCUSSION THREAD

**Discussion by adamphetamine** [Security-Onion-Solutions/securityonion#9581] [General] [answered]
ok apologies in advance for the dumb question, 
but I notice that some components like Elastic get upgraded a lot (2.3.170, 2.3.180, 2.3.200)
and some others not so much- 

Fleet (last update 2.3.130)
Wazuh (last update 2.1.0)

I'm not looking to put pressure on, just trying to understand -
Is it a policy thing? Or lack of testers, demand, magic or money falling from the sky*?
*me too

**Comment by InfosecGoon** [comment, marked as answer]
Generally, component updates are driven by things like new features, compatibility with other components, or security fixes. If a component is meeting its functional requirements and doesn't have any outstanding bugs or security issues, there's no compelling reason to spend the time testing and integrating a newer version.

For those particular components, there were some integration issues with FleetDM that we needed to work out, so that should be updated in the next release or two. Updating Wazuh to the current version would be a tremendous effort, so its being left as-is.

**Comment by adamphetamine** [comment]
Thanks for the detailed answer- that's exactly what I needed to know!
